swatch-system-conduit/src/main/spec/internal-organizations-sync-api-spec.yaml

```diff
@@ -0,0 +1,314 @@
+openapi: "3.0.2"
+info:
+  title: "internal-organizations-sync-api"
+  description: "REST interface for the internal-organization-sync service. Please note any deprecated APIs. Our current deprecation policy is to keep deprecated APIs around for at least 6 months."
+  version: 1.0.0
+paths:
+  /internal/rpc/syncOrg:
+    description: "Trigger a sync for a given Org ID"
+    post:
+      summary: "Sync organization for given org_id."
+      requestBody:
+        $ref: "#/components/requestBodies/OrgSyncRequestBody"
+      operationId: syncOrg
+      tags:
+        - internalOrganizations
+      responses:
+        '200':
+          description: "The request for syncing organization has completed successfully."
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: "#/components/schemas/DefaultResponse"
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/ResourceNotFound"
+        '500':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/InternalServerError"
+  /internal/rpc/syncAllOrgs:
+    description: "Trigger a sync for all Organizations"
+    post:
+      summary: "Sync all organizations."
+      operationId: syncFullOrgList
+      tags:
+        - internalOrganizations
+      responses:
+        '202':
+          description: "The request for syncing all organizations is successfully running."
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: "#/components/schemas/DefaultResponse"
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/ResourceNotFound"
+        '500':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/InternalServerError"
+  /internal/organizations-sync-list:
+    description: "Initiate altering database sync list by either adding or removing a given list of Org IDs"
+    parameters:
+      - name: org_ids
```

Comment:
We should define this as an array based query parameter.  Something like this should do the trick

```
- name: org_ids
  in: query
  required: true
  style: form
  explode: false
  schema:
    type: array
    items:
      type: string
```
See https://swagger.io/docs/specification/describing-parameters/#query-parameters and https://swagger.io/docs/specification/serialization/